docs/guides/cloud/introduction.mdx

```diff
@@ -4,153 +4,131 @@ sidebar_position: 10
 sidebar_label: Introduction
 ---
 
-:::tip
+[Cypress Cloud](https://on.cypress.io/cloud) is our enterprise-ready companion
+to the open-source Cypress app, allowing you to:
 
-<strong>
-  <Icon name="star" /> Get Started
-</strong>
+- [View past test results](#View-past-test-results) from your CI environment.
+- [Analyze and diagnose](#Analyze-and-diagnose-test-health) test failures.
+- [Reduce test time and save money](#Reduce-test-time--CI-costs) by smartly
+  orchestrating future runs across multiple machines.
+- [Integrate with your favorite developer tools](#Integrate-with-your-favorite-tools)
+  to help streamline your workflows.
+- [View CI runs and test health](#Cypress-app-integration) directly from the
+  Cypress app.
 
-To get started with Cypress Cloud,
-[sign up here](https://cloud.cypress.io/signup) for our
-[free plan](https://www.cypress.io/pricing) which comes with 3 users and 500
-monthly test results.
-
-:::
-
-[Cypress Cloud](https://on.cypress.io/cloud) is our enterprise-ready, web-based
-companion to the Cypress app. It gives you online access to your recorded test
-results, orchestrates test runs across multiple machines, provides rich
-analytics and diagnostics, and integrates those insights with your favorite
-tools.
-
-<!-- textlint-disable -->
-
-<DocsVideo src="https://vimeo.com/736212167" />
-
-<!-- textlint-enable -->
-
-:::info
-
-##### <Icon name="graduation-cap" /> Real World Example <Badge type="success">New</Badge>
-
-The Cypress
-[Real World App (RWA)](https://github.com/cypress-io/cypress-realworld-app)
-leverages [Cypress Cloud in CI](https://cloud.cypress.io/projects/7s5okt) to
-test over 300 test cases in parallel across 25 machines, multiple browsers,
-multiple device sizes, and multiple operating systems.
-
-Check out the <Icon name="github" />
-[Real World App in Cypress Cloud](https://cloud.cypress.io/projects/7s5okt).
-
-:::
+[Get started with Cypress Cloud](/guides/cloud/getting-started) for free now, or
```

Comment:
I assume the previous treatment with the very specific _with 3 users and 500
monthly test results._ verbiage was a result of CS getting this question a lot. might be worth confirming with Shawn.

Reply:
I moved this over to the getting started guide.